src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import Select, { Props, StylesConfig } from 'react-select';
+
+import { SelectArrow } from 'assets/svg';
+
+type Option = { label: string; value: string; image?: string };
+
+const styles: StylesConfig<Option> = {
+  container: (styles) => ({
+    ...styles,
+
+    '.customOption': {
+      height: '32px',
+      display: 'flex',
+      alignItems: 'center',
+
+      '&-img': {
+        marginRight: '9px',
+        width: '32px',
+        height: '32px',
+      },
+
+      '&-label': {
+        color: '#061953',
+        lineHeight: '21px',
+        fontWeight: 700,
+        fontSize: 14,
+      },
+    },
+  }),
+  control: (styles) => ({
+    ...styles,
+    ':hover': {
+      cursor: 'pointer',
+    },
+    backgroundColor: '#fcfcfc',
+    padding: '12px 16px',
+    boxSizing: 'border-box',
+    height: '56px',
+    border: '1px solid rgba(22, 32, 44, 0.08)',
+    borderRadius: '8px',
+    boxShadow: '0px 10px 40px rgba(6, 25, 83, 0.08)',
+  }),
+  indicatorSeparator: () => ({ display: 'none' }),
+  indicatorsContainer: (styles) => ({
+    ...styles,
+    width: 24,
+    justifyContent: 'center',
+  }),
+  valueContainer: (styles) => ({
+    ...styles,
+    padding: 0,
+    marginBottom: 'auto',
+  }),
+  menuList: (styles) => ({
+    ...styles,
+    padding: '10px 0',
+    maxHeight: '338px',
+    '::-webkit-scrollbar': {
+      background: 'rgba(0,0,0,0)',
+      width: '4px',
+    },
+    '::-webkit-scrollbar-thumb': {
+      borderRadius: '4px',
+      background: 'rgba(109, 118, 146, 0.5)',
+    },
+  }),
+  menu: (styles) => ({
+    ...styles,
+    borderRadius: '8px',
+    border: '1px solid rgba(22, 32, 44, 0.08)',
+    background: '#fbfbfb',
+    paddingRight: '3px',
+  }),
+  option: (styles) => ({
+    ...styles,
+    background: '#fbfbfb',
+    padding: '10px 16px',
+    ':hover': {
+      background: '#DEEBFF',
+      cursor: 'pointer',
+    },
+  }),
+};
+
+export const Dropdown = (props: Props<Option, false, any>): JSX.Element => {
+  return (
+    <Select
+      components={{ DropdownIndicator: () => <SelectArrow /> }}
+      isSearchable={false}
+      options={props.options}
+      styles={styles}
+      {...props}
+      formatOptionLabel={(asset) => (
```

Comment:
no need to bind naming to asset, make component more generic

```suggestion
      formatOptionLabel={(option) => (
```

Reply:
Done